### PR TITLE
Update README documentation from release branch

### LINKED ISSUE #57: Master inconsistent with regional_workflow/develop
With the update of regional_workflow with [#PR 357,](https://github.com/NOAA-EMC/regional_workflow/pull/357) the master branch of SRWeather App no longer works. It appears that PR #45 merged to master would fix the problem of sourcing the README txt file. It is not clear to me if there are other inconsistencies between master here and develop in regional_workflow.

### PR DESCRIPTION
**Pull request by mkavulich**
## DESCRIPTION OF CHANGES: 
Bring over README documentation from release branch since it is now sourced by the develop branch of regional_workflow (since commit https://github.com/NOAA-EMC/regional_workflow/commit/0711f0eb6d9ac51a946fe4cc9814795abb63b951). Therefore these files are needed for continued development on the regional_workflow repository from develop.

## TESTS CONDUCTED: 
None yet.

## ISSUE (optional): 
Partially resolves #57

### CODE REVIEW

**Comment by mkavulich** [comment]
I'm aware that ideally this issue would be resolved by cherry-picking all needed commits from the release branch, but that would require extensive testing due to various changes made in the interim, and this needs to be fixed urgently for further development purposes.

**Review by christinaholtNOAA — CHANGES_REQUESTED** [review]
My tests on Jet are failing on the build. l will need some additional help with this test.

So far, I have tried to point to the develop branch of ufs-weather-model after failures due to a severely reduced set of CCPP packages in the release branch. And am now failing with issues seemingly related to inline post. Will follow up in person....

File: docs/INSTALL [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# in this directory. These README files give instructions assuming a bash or ksh login shell, for
+# csh and tcsh users you will have to modify the commands for setting envronment variables.
+
+# Supported CMake flags:
+#  -DCMAKE_INSTALL_PREFIX Location where the bin/ include/ lib/ and share/ directories containing
+#                         the various components of the SRW App will be created. Recommended value
+#                         is "..", one directory up from the build directory
+#  -DCCPP_SUITES          A comma-separated list of CCPP suites to build with the UFS weather 
+#                         model. See the User's Guide for a full list of available suites.
+
```
[the remark this answers is not in the document]
Comment (christinaholtNOAA) [reply docs/INSTALL:26]: Well, this was me being dumb...please disregard.

**Review by JulieSchramm — APPROVED** [review]

**Comment by JeffBeck-NOAA** [comment]
@mkavulich @christinaholtNOAA @gsketefian, the default Externals.cfg for the develop branch of ufs-srweather-app needs to be changed to grab the develop branch of ufs-weather-model, not release/public-v2.  My first build error was related to missing CCPP suites.  @mkavulich, would you mind making that switch and pushing it to this PR?  Now, I'm getting the following when building ufs-weather-model:

/lfs4/BMC/wrfruc/beck/ufs-srweather-app_Mike/src/ufs_weather_model/FV3/fv3_cap.F90(1157): error #6632: Keyword arguments are invalid with
         call ESMF_VMEpochEnter(epoch=ESMF_VMEpoch_Buffer, rc=rc)
--------------------------------^

**Comment by christinaholtNOAA** [comment]
@JeffBeck-NOAA The top of develop does not work, either.

**Comment by JeffBeck-NOAA** [comment]
> @JeffBeck-NOAA The top of develop does not work, either.

Yes, the modules in the README files are currently based off the release branches for all repositories.  A set of develop branch repo hashes (could be the HEAD for all) will need to be defined in the Externals.cfg file, along with a set of compatible modules for those branches in the README files.  Once that is set up, then the compile should work.

**Comment by mkavulich** [comment]
Since this quick fix turned out to be a dead end, I am closing this PR to open a new, better one merging commits from the release branch (to go in after #63)

**Pull request closed without merging**